src/libraries/System.Text.Json/src/System/Text/Json/Serialization/MemberAccessor.cs

```diff
@@ -29,9 +29,9 @@ protected MethodInfo ImmutableCollectionCreateRangeMethod(Type constructingType,
             return createRangeMethod.MakeGenericMethod(elementType);
         }
 
-        protected MethodInfo ImmutableDictionaryCreateRangeMethod(Type constructingType, Type elementType)
+        protected MethodInfo? ImmutableDictionaryCreateRangeMethod(Type constructingType, Type elementType)
```

Comment:
Similarly here, don't return nullable.
```suggestion
        protected MethodInfo ImmutableDictionaryCreateRangeMethod(Type constructingType, Type elementType)
```